tokio/src/time/clock.rs

```diff
@@ -132,8 +132,7 @@ cfg_test_util! {
             }
         }
 
-        // TODO: delete this. Some tests rely on this
-        #[cfg(all(test, not(loom)))]
+        #[cfg(all(not(loom)))]
```

Comment:
I'm guessing that with the removal of the `TODO`, this working now?

Reply:
I did another pull request related to this. The problem was pausing and resuming time would yield an Instance that might be less than a previous Instant, which breaks a guarantee of std::time::Instant. It might be relevant as well: https://github.com/tokio-rs/tokio/pull/2253